internal/services/containerapps/helpers/container_apps.go

```diff
@@ -2364,8 +2408,16 @@ func ContainerAppStartupProbeSchema() *pluginsdk.Schema {
 					Type:         pluginsdk.TypeInt,
 					Optional:     true,
 					Default:      3,
-					ValidateFunc: validation.IntBetween(1, 10),
-					Description:  "The number of consecutive failures required to consider this probe as failed. Possible values are between `1` and `10`. Defaults to `3`.",
+					ValidateFunc: validation.IntBetween(1, 30),
+					Description:  "The number of consecutive failures required to consider this probe as failed. Possible values are between `1` and `30`. Defaults to `3`.",
+				},
+
+				"success_count_threshold": {
+					Type:         pluginsdk.TypeInt,
+					Optional:     true,
+					Default:      1,
+					ValidateFunc: validation.IntBetween(1, 1),
```

Comment:
Likewise here, is there a point in exposing this if it's default is `1` and can only be set to `1` anyway?

Reply:
No, it should be exactly 1. I added it as it's present in UI an in ContainerAppReadinessProbeSchema. Should I remove it?
<img width="566" alt="image" src="https://github.com/user-attachments/assets/2ffbe2dc-c691-49f8-9510-0bb49da03d9b">

Reply:
Removed where I added it. But kept it in ContainerAppReadinessProbeSchema to not create a breaking change